README.md

```diff
@@ -205,45 +205,6 @@ if(account != null){
 ## ProGuard
 MSAL uses reflection and generic type information stored in `.class` files at runtime to support various persistence and serialization related functionalities. Accordingly, library support for minification and obfuscation is limited. For reliability, we recommend developers exclude the Microsoft package from ProGuard.
```

Comment:
Do we still need this bit in README?

Reply:
Only if you (or others) think it provides useful information as to why the proguard config is the way it is.

I have no strong feelings about including this if it's just noise

Reply:
I think we should remove it (or at least remove the last statement) because it seems like we are still asking developers to exclude MSAL classes within their ProGaurd file, but I believe developers no longer need to that since we have included our own ProGaurd file, right?

Reply:
Updated in 79d2152a68852c70c8cb7a4cf72dd6377d32524f

How does that sound now?

Reply:
Yeah that sounds good!